### ISSUE TITLE
Use Targetplatform for the build

### ISSUE THREAD

**Issue by hendrikstill** [vogellacompany/com.vogella.vde#9]
First assure if it is really necessary to use a targetplatform

**Comment by hendrikstill** [comment]
I'm still not 100% sure if we need to use a specific target platform.

I guess for a stable release this makes sens to have a static dependency resolving (not at build time).
For unstable releases like our voclipse luna build, we would have to manually update/syncronise this file, which could be done by tycho (like the current situation, where tycho resolves the target platform from the the given p2 sites).

For me the target definition is a file which defines a specific environment of bundles, which a developer is promised on the target system. This allows to compile, launch and verify the dependencies against the target system. 
But what we are not developing a for a target system, we are building the target system, so I personally see no need here. 
@vogella  or do I still don't get it? I'm still not 100% sure about this. 

Resource:
http://help.eclipse.org/kepler/index.jsp?topic=%2Forg.eclipse.pde.doc.user%2Fguide%2Ftools%2Ftarget_shared%2Fedit_target_wizard.htm

**Comment by vogella** [comment]
I might be wrong here but the target file is IMHO only a container for
describing p2 update sides. So in theory you can move your build from Luna
to Kepler without touching the pom.
Am 09.12.2013 18:42 schrieb "Hendrik S" notifications@github.com:

**Comment by hendrikstill** [comment]
I've added a target platform description to the luna build. The development for this takes place in a seperated branch "target_platform" [1]

The target platform description has been defined by the "Target Editor" so currently strict version numbers are given.
e.g.: org.eclipse.jdt.java8patch.feature.group version="1.0.0.v20140103-0106_BETA_JAVA8"
Which is not optimal for our purpose. With this workflow the resolving of the target platform is done at designtime. So to upgrade the bundle version numbers, we have to manually update this target file.  

[1]https://github.com/vogellacompany/com.vogella.vde/tree/target_platform

**Comment by vogella** [comment]
Can the versions left out?
Am 03.01.2014 12:20 schrieb "Hendrik S" notifications@github.com:

**Comment by hendrikstill** [comment]
Yes this is possible but I've currently found no documentation about this "feature".
